CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## XX.XX.XX - 20XX-XX-XX
 
+*[ADDED][9158](https://github.com/stripe/stripe-android/pull/9158) Add `requireCvcRecollection` param to `IntentConfiguration`
```

Comment:
We should mention this is still in private beta, it's a breaking API change, and how to migrate.

Reply:
Hmm, we have the migration docs ready but they're not deployed. Should I remove the changelog addition and add once we're ready to GA?

Reply:
We need to mention the breaking change (removing the callback). Ideally we point to the migration guide as well.

Reply:
Added verbiage about breaking change and private beta. Will update to point to migration guide if we have approval by the next release